### DISCUSSION TITLE
我该怎样才能收到激活码，续费支付都一段时间了

### DISCUSSION THREAD

**Discussion by kidokogills** [getlantern/lantern-forum-cn#837] [Q&A]
![微信图片_20240829232805](https://github.com/user-attachments/assets/2c545900-3c0e-4013-b7cf-f9dfcfe6a7f8)
@Mike777Khoo 续费支付已有一段时间了，账号支付单据都给了，请问什么时候可以获得激活码，都付款了，就算有情况，也不至于没有任何回复吧，如果激活码发不了邮箱firekyd@163.com，请发到firekydchan@gmail.com

**Comment by kidokogills** [comment]
我是真搞不懂，前面一堆骂的喊退款的，你们回复得那么积极，付了款好好说话的你们就视而不见，理都不理，是打算拖够一个月无法退款直接把钱吞了？一个激活码的事情，能拖这么长时间？

**Comment by MaxMKMax** [comment]
@kidokogills 您好. 我帮您再追问支付团队一下. 非常抱歉让您久等了!

**Comment by MaxMKMax** [comment]
![image](https://github.com/user-attachments/assets/ca998e3f-fd4b-4d03-8b26-4e722764c995)
上个星期支付团队已经解决您的问题了啊. 您已经是专业版了. 您是遇到什么问题吗?

**Reply by kidokogills** [reply]
我没有激活码

**Reply by kidokogills** [reply]
一直都是免费版状态

**Reply by MaxMKMax** [reply]
@kidokogills 8 月 30 日之前问题就解决了. 您可以看截图, 我们的工作聊天记录.

**Reply by MaxMKMax** [reply]
@kidokogills 我刚刚又帮您追问啦. 请放心. 有答复了马上联系您.
![image](https://github.com/user-attachments/assets/59b14c50-5588-4aed-bd75-65be674af197)

翻译:
8.firekyd@163.com
回到8月30日，案例6。我想你已经说过它已经是专业版了。但今天用户仍然反馈无法登录pro。无法获取激活码。不确定发生了什么问题。
https://github.com/getlantern/lantern-forum-cn/discussions/837
用户：如果激活码无法发送至邮箱firekyd@163.com，请发送至firekydchan@gmail.com

**Reply by kidokogills** [reply]
@Mike777Khoo 无论是电脑端还是手机端，都需要我输入激活码，但问题是，我一直没收到任何邮件，两个邮箱都没有，如果说已经解决，那目前情况就是我缺一个激活码

**Comment by MaxMKMax** [comment]
@kidokogills 您看. 我们是有认真在帮用户解决问题的啊. 请您放心, 谢谢您的耐心等待.

明白了. 已经在帮您拿激活码. 我在等待他们回复我

**Reply by kidokogills** [reply]
好的，感谢，我也在等

**Reply by kidokogills** [reply]
如果激活码已发送烦请告知一下，我可以确认是不是邮箱无法接收的问题，如果是邮箱问题，看能不能直接发这里，麻烦了

**Reply by MaxMKMax** [reply]
@kidokogills 支付团队已经帮您更换邮箱了. 您应该可以收到激活码了. 您用新的邮箱登陆专业版就可以了. 
换成这个: 
firekydchan@gmail.com

**Reply by MaxMKMax** [reply]
@kidokogills 如果还有什么问题请您马上联系我. 我再帮您跟进.

**Reply by kidokogills** [reply]
好的，感谢，我在刷新收件箱，暂时没收到新邮件

**Reply by kidokogills** [reply]
@Mike777Khoo 你好，请问邮件是系统发送还是团队人工的，如果是人工，我能否理解为最快下周才能收到邮件？

**Reply by kidokogills** [reply]
@Mike777Khoo  你好，我到现在都没收到激活码的邮件，请问什么时候会发送

**Reply by kidokogills** [reply]
@Mike777Khoo 你好，目前还没有收到激活码的邮件，请问什么时候发送？

**Reply by kidokogills** [reply]
> @kidokogills如果还有什么问题请您立即联系我。 我再帮助您探索。

@Mike777Khoo 到目前为止，仍还没有收到激活码的邮件，能否跟进一下，请问什么时候发送？真的，要登录账号，缺一个激活码
